src/component/graph/Graph.ts

```diff
@@ -12,8 +12,49 @@ mxGraph.prototype.edgeLabelsMovable = false;
 mxGraph.prototype.cellsLocked = true;
 mxGraph.prototype.cellsSelectable = false;
 
+class MySvgCanvas2D extends mxSvgCanvas2D {
```

Comment:
```suggestion
class BpmnVisuSvgCanvas2D extends mxSvgCanvas2D {
```

Reply:
:question: what do you thing about adding a comment about what this implem provides comparing to the original one?

Reply:
done